tests/acceptance/features/apiContract/getSpaces.feature

```diff
@@ -0,0 +1,64 @@
+@api @skipOnOcV10
+Feature: GET drives/spaces
+  check that the GET drives/spaces response contains all the relevant value
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario: user requests own space information (Personal space)
+    When user "Alice" lists all available spaces via the GraphApi
+    Then the HTTP status code should be "200"
+    And the json responded should contain a space "Alice Hansen" owned by "Alice" with these key and value pairs:
+      | key               | value                            |
+      | driveAlias        | personal/alice                   |
+      | driveType         | personal                         |
+      | id                | %space_id%                       |
+      | name              | Alice Hansen                     |
+      | owner@@@user@@@id | %user_id%                        |
+      | quota@@@state     | normal                           |
+      | root@@@id         | %space_id%                       |
+      | root@@@webDavUrl  | %base_url%/dav/spaces/%space_id% |
+      | webUrl            | %base_url%/f/%space_id%          |
+
+
+  Scenario: user requests own space information (Shares space)
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Brian" has created folder "folder"
+    And user "Brian" has shared folder "folder" with user "Alice" with permissions "31"
+    And user "Alice" has accepted share "/folder" offered by user "Brian"
+    When user "Alice" lists all available spaces via the GraphApi
+    Then the HTTP status code should be "200"
+    And the json responded should contain a space "Shares" with these key and value pairs:
+      | key               | value                            |
+      | driveAlias        | virtual/shares                   |
+      | driveType         | virtual                          |
+      | id                | %space_id%                       |
+      | name              | Shares                           |
+      | quota@@@remaining | 0                                |
+      | quota@@@state     | exceeded                         |
```

Comment:
Correct. Still an issue. Should be fixed long-term https://github.com/owncloud/ocis/issues/4472